js/pricing.js

```diff
@@ -9,17 +12,44 @@ $(
     function () {
         const $confirmPersonalInformation = $('#confirm-personal-information');
         const $confirmDevelopmentAgreement = $('#confirm-development-agreement');
-        const $disabledButton = $('#order-now-btn');
+        const $orderButton = $('#order-now-btn');
+        const $redirect = $('#redirect');
```

Comment:
Is it a button? Or a block?

I'd suggest considering some better naming. Right now you just duplicate the selector and it has very little value.